jdaviz/cli.py

```diff
@@ -16,14 +16,11 @@
 
 @click.version_option(__version__)
 @click.command()
+@click.argument('layout',
+                nargs=1,
+                type=click.Choice(['cubeviz', 'specviz', 'mosviz', 'imviz'],
+                                  case_sensitive=False))
 @click.argument('filename', nargs=1)
-@click.option('--layout',
```

Comment:
Is there no way to deprecate this first rather than straight out removing it? Maybe this has been discussed while I am out and I missed it.

Reply:
I'm open to any suggestions.  Its definitely possible to still support `--layout` by renaming the positional argument to something else and checking to make sure if `--layout` is provided, its the same as the positional argument... but since the positional one is required, that seems like overkill.  If we switch the order so that the positional argument for layout is after the filename, then I think something like that becomes much more sensible, but it feels less natural to me to have `jdaviz /path/to/file cubeviz` than `jdaviz cubeviz path/to/file`.

Reply:
I was actually thinking one step more backward, as in holding off `layout` as positional, but issuing a message for `--layout`, which means everyone using the CLI will see this message for at least one release, before the API changes. Although given the small amount of users we have, maybe this is overkill?

Reply:
That's definitely an option too... I'm just not sure how to make a message visible enough (for any use that is just launching from habit, a print to the terminal might easily go unnoticed).  I did just test issuing the old syntax in this PR though, and it gives the following (which isn't maybe perfectly ideal, but at least does kind of point the user in the right direction to noticing the change?):

```
Usage: jdaviz [OPTIONS] {cubeviz|specviz|mosviz|imviz} FILENAME
Try 'jdaviz --help' for help.

Error: No such option: --layout
```

Reply:
Maybe what you have is good enough.